composer.json

```diff
@@ -58,6 +58,7 @@
         "symfony/console": "^4.3",
         "symfony/css-selector": "^4.3",
         "symfony/dependency-injection": "^4.3",
+        "symfony/debug": "^4.3",
```

Comment:
required by the ExceptionController: https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/WebsiteBundle/Controller/ExceptionController.php#L17